index.html

```diff
@@ -1126,517 +1231,271 @@ <h4>Time expressions</h4>
-              <td>prohibited</td>
-              <td></td>
-            </tr>
-            <tr>
-              <td><code>#writingMode-horizontal-rl</code></td>
-              <td>prohibited</td>
-              <td></td>
-            </tr>
-            <tr>
-              <td><code>#writingMode-horizontal</code></td>
-              <td>prohibited</td>
-              <td></td>
-            </tr>
-            <tr>
-              <td><code>#zIndex</code></td>
-              <td>prohibited</td>
-              <td></td>
-            </tr>
-          </tbody>
-        </table>
-      </section> <!-- features -->
 
       <section>
         <h4>Layout</h4>
-        <p>This specification does not put additional constraints on the layout and rendering <a>features</a> defined in [[!ttml-imsc1.2]].</p>
+        <p>This specification does not put additional constraints on the layout and rendering <a>Features</a> defined in [[!ttml-imsc1.2]].</p>
         <div class="note">Layout of the paragraphs may rely on the default TTML region (i.e. if no <code>layout</code> is used in the <code>head</code> element) or may be explicit by the use of the <code>region</code> attribute, if a <code>region</code> element is defined in a <code>layout</code> element in the <code>head</code> element.</div>
       </section>  <!-- Layout -->
 
     </section> <!-- Profile Constraints -->
 
   <section id="conformance">
+    <section id="content-profile-conformance-section">
+      <h3>Conformance of <a>DAPT Scripts</a></h3>
+      <p>Conformant <a>DAPT Scripts</a> are [[ttml2]] <a>Document Instances</a> that
+        conform to the normative provisions of this specification,
+        expressed using the vocabulary of [[ttml2]] in the
+        <dfn data-cite="ttml2#terms-content-profile">content profile</dfn> defined
+        in <a href="#profiles-section"></a>.</p>
+    </section> <!-- content-profile-conformance-section -->
+    <section id="processor-profile-conformance-section">
+      <h3>Conformance of DAPT Processors</h3>
+      <p>Conformant DAPT Processors are [[ttml2]]
+        <dfn data-cite="ttml2#terms-content-processor">content processors</dfn> that
+        conform to the normative provisions of this specification,
+        expressed using the vocabulary of [[ttml2]] in the
+        <dfn data-cite="ttml2#terms-processor-profile">processor profile</dfn> defined
+        in <a href="#profiles-section"></a>.</p>
+    </section> <!-- processor-profile-conformance-section -->
+  </section> <!-- conformance -->
+
+  <section id="index" class="appendix">
+    <!-- All the terms will magically appear here -->
+  </section>
+
+  <section id="extensions-section" class="appendix">
+    <h2>Extensions</h2>
+    <section id="extensions-general">
+      <h3>General</h3>
+      <p>The following sections define extension designations,
+        expressed as relative URIs (fragment identifiers)
+        relative to the <a>DAPT Extension</a> Namespace base URI.
+      </p>
```

Comment:
```suggestion
      These extension designations are solely defined to permit the formal definition of DAPT
      as profile of [[ttml2]] as in <a href="#profiles-section"></a>.
      </p>
```

Reply:
I would go even further and say something like: 
* DAPT content authors should not care about Annex B and C
* DAPT processor implementers should not care about Annex B and C, unless they base their implementation on a generic TTML2 processor that needs to be tailored for TTML2.